service/pom.xml

```diff
@@ -82,13 +76,27 @@
             <version>2.3.9</version>
         </dependency>
 
+        <!-- JSON -->
+        <dependency>
+            <groupId>com.fasterxml.jackson.datatype</groupId>
+            <artifactId>jackson-datatype-jsr310</artifactId>
+            <version>2.6.1</version>
```

Comment:
Very minor, but the latest stable version of this module is 2.8.3. Should we use that instead maybe?